src/components/change-requests/change-request-details/change-request-details/review-notes/review-notes.tsx

```diff
@@ -3,20 +3,37 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { User } from 'utils';
-import { fullNamePipe } from '../../../../../shared/pipes';
+import { datePipe, fullNamePipe } from '../../../../../shared/pipes';
 import PageBlock from '../../../../shared/page-block/page-block';
 
 interface ReviewNotesProps {
   reviewer?: User;
   reviewNotes?: string;
+  dateReviewed: Date;
```

Comment:
Make sure to update the tests, especially after adding another field to the props of a component.